docker/tests/Dockerfile

```diff
@@ -67,9 +66,12 @@ RUN rm -rf build log install src/lrauv
 # Use current version of the code
 COPY . src/lrauv
 
-# Build image
+# Build workspace
 RUN [ "/bin/bash" , "-c" , "colcon build --cmake-args='-DBUILD_TESTING=true'" ]
 
+COPY docker/entrypoint.bash /
+# setup ignition environment
+ENTRYPOINT ["/entrypoint.bash"]
+
 # Run tests
-RUN [ "/bin/bash" , "-c" , ". ~/lrauv_ws/install/setup.sh; colcon test --event-handlers console_direct+" ]
-RUN [ "/bin/bash" , "-c" , ". ~/lrauv_ws/install/setup.sh; colcon test-result" ]
+CMD [ "/bin/bash" , "-c" , "colcon test --event-handlers console_direct+; colcon test-result" ]
```

Comment:
I don't see the test output on GitHub actions, did it run and the output got swallowed?

Reply:
Ahh, the actions aren't actually running the image, only building the image. I'll add it to the action.